src/Microsoft.DotNet.PlatformAbstractions/Native/PlatformApis.cs

```diff
@@ -112,11 +112,29 @@ private static DistroInfo LoadDistroInfo()
                         result.VersionId = line.Substring(11).Trim('"', '\'');
                     }
                 }
-                return result;
+
+                return NormalizeDistroInfo(result);
             }
             return null;
         }
 
+        // For some distros, we don't want to use the full version from VERSION_ID. One example is
+        // Red Hat Enterprise Linux, which includes a minor version in their VERSION_ID but minor
+        // versions are backwards compatable.
+        //
+        // In this case, we'll normalized RIDs like 'rhel.7.2' and 'rhel.7.3' to a generic
+        // 'rhel.7'. This brings RHEL in line with other distros like CentOS, Ubuntu or Debian which
```

Comment:
I agree that the comment should be tweaked. Most distributions with a minor version are not backwards compatible. Or if they are backwards compatible, they do not use the minor version. RHEL is a bit of an exception where it is backwards compatible and uses a minor version (which doesn't affect compat).

Ubuntu is a bit of a strange case itself. It uses the dot (`.`) not as a separator between the major and minor versions but as a date-separator. `14.04` indicates that the release was done in 04/2014. Different minor versions (aka the 2014/04 and 2014/10 releases) are not backwards compatible with each other.